### PR TITLE
Updated Pase Cookie Example check if values have double quotes

### PR DESCRIPTION
**Pull request by CreaTorAlexander**
Hey, 
I've added a check if the values contain double quotes. 
When this is the case then I slice the quotes away.

### CODE REVIEW

**Comment by dostiharise** [comment]
@CreaTorAleXander, 

Nice, I wish I could approve and merge. 

You may need to get some attention of the maintainers. Try tagging them.

File: snippets/parseCookie.md [hunk 1]
```diff
@@ -15,12 +15,21 @@ const parseCookie = str =>
     .split(';')
     .map(v => v.split('='))
     .reduce((acc, v) => {
-      acc[decodeURIComponent(v[0].trim())] = decodeURIComponent(v[1].trim());
+      // check if values have double quotes
+      // when yes then slice these double quotes away
+    if(v[1].trim().startsWith("\"") && v[1].trim().endsWith("\"")){
+        acc[decodeURIComponent(v[0].trim())] = decodeURIComponent(v[1].slice(1, v[1].length - 2));
```
Comment (Li9x1) [inline snippets/parseCookie.md:21]: how about using regex? something like this.. 
`v[1].trim().replace(/^"|"$/g, '')`

Comment (CreaTorAlexander) [reply snippets/parseCookie.md:21]: Yes, that's a good solution.
But I have to say I like the way with startsWith and endsWith a bit more.
If you think .replace would make more sense then I can update it :+1:

Comment (DrSameerHealth) [reply snippets/parseCookie.md:21]: 👍

**Comment by Chalarangelo** [comment]
Closing as the original issue was not valid.

**Pull request closed without merging**